Library/OcApfsLib/OcApfsConnect.c

```diff
@@ -392,7 +392,7 @@ ApfsStartDriver (
 
 STATIC
 EFI_STATUS
-ApfsConnectDevice (
+OcApfsConnectDevice (
```

Comment:
This is a private unexported (STATIC) function. Usually we do not give private functions an Oc prefix. In my opinion, leave it just Apfs. Similarly we give Internal prefix to private exported (non-STATIC) functions.

Reply:
Got it on the static function naming although not consistently applied throughout the project.

Reply:
Any steer on the unused `ApfsCheckOpenCoreScanPolicy` function?

Reply:
It is used, at least in master.

Reply:
Found the usage. Cheers.
Wil push update later.